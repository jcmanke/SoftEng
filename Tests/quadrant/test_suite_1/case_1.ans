The point lies on the y axis 
