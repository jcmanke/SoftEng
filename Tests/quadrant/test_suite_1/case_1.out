The point lies on the y axis 
The point lies in the second quadrant 
